PURL-SPECIFICATION.rst

```diff
@@ -482,22 +488,22 @@ pairs some of which may not be normalized:
 - **qualifiers**: the ``qualifiers`` corresponding to this ``purl`` as an object of
   {key: value} qualifier pairs.
 - **subpath**: the ``subpath`` corresponding to this ``purl``.
-- **is_invalid**: a boolean flag set to true if the test should report an
+- **is_invalid**: a boolean flag set to true if the test SHOULD report an
```

Comment:
This is incorrect. The test MUST report an error if the error is related to general PURL problems or if the error is related to a package type problem for a package type that is supported by the implementation being tested.